apps/molgenis-components/src/components/tables/TableSimple.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div class="table-responsive">
+    <table class="table table-bordered" :class="{ 'table-hover': tableHover }">
+      <thead>
+        <tr>
+          <th scope="col" style="width: 1px" v-if="hasColheader">
+            <slot name="colheader" />
+          </th>
+          <th v-for="col in columns" :key="col" scope="col">
+            {{ col }}
+          </th>
```

Comment:
minor thing, but I'd prefer writing out variable names. In this case 'column' or 'columnName'